File: app/src/main/java/mishka/rssreader/data/model/Channel.java
package mishka.rssreader.data.model;

import org.simpleframework.xml.ElementList;
import org.simpleframework.xml.Root;

import java.util.List;

@Root(name = "channel", strict = false)
public class Channel {
    @ElementList(name = "item", required = false, inline = true)
    private List<RssItem> items;

    public List<RssItem> getItems() {
        return items;
    }
}

File: app/src/main/java/mishka/rssreader/data/model/Channel.kt
package mishka.rssreader.data.model;

import org.simpleframework.xml.ElementList;
import org.simpleframework.xml.Root;


@Root(name = "channel", strict = false)
class Channel{
    @field:ElementList(name = "item", required = false, inline = true) var items: List<RssItem>? = null
}